mozetl/shield/containers.py

```diff
@@ -0,0 +1,34 @@
+from pyspark.sql.types import StringType, LongType
+
+from ..basic import convert_pings, DataFrameConfig
+from .utils import testpilot_etl_boilerplate
+
+
+def transform_pings(sqlContext, pings):
```

Comment:
That should work. [This notebook](https://gist.github.com/harterrt/fb9cdc217da4e0410556c9190132d0a5) is scheduled to run every day on ATMO. It would be trivial to update the notebook to run both the `testpilot_etl_job` and `shield_etl_job` instead. 

The other option would be wrapping these two function in a third `etl_job` which would require no modification elsewhere.